<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createOwnGame</class>
 <widget class="QWidget" name="createOwnGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="__wrapper" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>65</number>
    </property>
    <property name="leftMargin">
     <number>460</number>
    </property>
    <property name="topMargin">
     <number>65</number>
    </property>
    <property name="rightMargin">
     <number>460</number>
    </property>
    <property name="bottomMargin">
     <number>155</number>
    </property>
    <item>
     <widget class="QLabel" name="__title_text">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1000</width>
        <height>95</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Marmelad</family>
        <pointsize>60</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выберите изображение</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="__content_box" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1171</width>
        <height>700</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>28</number>
       </property>
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>32</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>31</number>
       </property>
       <item>
        <widget class="QWidget" name="__preview_box" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>637</width>
           <height>637</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <widget class="QLabel" name="__preview_label">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>637</width>
            <height>637</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>637</width>
            <height>637</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="__v_layout_1">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="__v_layout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>134</number>
           </property>
           <property name="bottomMargin">
            <number>106</number>
           </property>
           <item>
            <widget class="QWidget" name="__w_load_images" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>450</width>
               <height>183</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>43</number>
              </property>
              <property name="leftMargin">
               <number>19</number>
              </property>
              <property name="topMargin">
               <number>33</number>
              </property>
              <property name="rightMargin">
               <number>31</number>
              </property>
              <property name="bottomMargin">
               <number>30</number>
              </property>
              <item>
               <widget class="QLabel" name="__li_text">
                <property name="minimumSize">
                 <size>
                  <width>237</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Marmelad</family>
                  <pointsize>35</pointsize>
                 </font>
                </property>
                <property name="contextMenuPolicy">
                 <enum>Qt::NoContextMenu</enum>
                </property>
                <property name="text">
                 <string>Загрузить</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="__li_images">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="../../sources/project_sources.qrc">:/images/load_images.png</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="_v_layout_3">
           <property name="spacing">
            <number>34</number>
           </property>
           <item>
            <widget class="QPushButton" name="__button_next">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>450</width>
               <height>90</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Marmelad</family>
               <pointsize>40</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>Продолжить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="__button_back">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>450</width>
               <height>90</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Marmelad</family>
               <pointsize>40</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>Вернуться</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../sources/project_sources.qrc"/>
 </resources>
 <connections/>
</ui>
